src/main/java/uk/gov/companieshouse/api/accounts/transformer/CreditorsAfterOneYearTransformer.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.companieshouse.api.accounts.transformer;
+
+import org.springframework.beans.BeanUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.model.entity.notes.creditorsafteroneyearentity.CreditorsAfterOneYearDataEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.notes.creditorsafteroneyearentity.CreditorsAfterOneYearEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.notes.creditorsafteroneyearentity.CurrentPeriodEntity;
+import uk.gov.companieshouse.api.accounts.model.entity.notes.creditorsafteroneyearentity.PreviousPeriodEntity;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.creditorsafteroneyear.CreditorsAfterOneYear;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.creditorsafteroneyear.CurrentPeriod;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.creditorsafteroneyear.PreviousPeriod;
+
+
+@Component
+public class CreditorsAfterOneYearTransformer implements GenericTransformer<CreditorsAfterOneYear, CreditorsAfterOneYearEntity>  {
+
+    @Override
+    public CreditorsAfterOneYearEntity transform(CreditorsAfterOneYear rest) {
+
+        CreditorsAfterOneYearDataEntity creditorsAfterOneYearDataEntity = new CreditorsAfterOneYearDataEntity();
+        CreditorsAfterOneYearEntity creditorsAfterOneYearEntity = new CreditorsAfterOneYearEntity();
+
+        BeanUtils.copyProperties(rest, creditorsAfterOneYearDataEntity);
+
+        if (rest.getCurrentPeriod() != null) {
+            CurrentPeriodEntity currentPeriodEntity = new CurrentPeriodEntity();
+            BeanUtils.copyProperties(rest.getCurrentPeriod(), currentPeriodEntity);
+            creditorsAfterOneYearDataEntity.setCurrentPeriodEntity(currentPeriodEntity);
+        }
+
+        BeanUtils.copyProperties(rest, creditorsAfterOneYearDataEntity);
```

Comment:
I think this second ` BeanUtils.copyProperties(rest, creditorsAfterOneYearDataEntity);` is not needed